docs/manage/config-policies.rst

```diff
@@ -0,0 +1,263 @@
+or CLI.
+
+WebUI
+=====
+
+Administrators can set Config Policies at both the cluster and workspace levels.
+
+.. tabs::
+
+   .. tab::
+
+      Cluster Config Policy
+
+      To set configuration policies at the cluster level:
+
+      #. Sign in to the Determined WebUI as a cluster administrator.
+      #. Navigate to **Config Policies**.
+      #. Choose **Experiments** or **Tasks** to display the editable configuration file.
+      #. Define the policies, then click **Apply**.
+      #. Confirm you want to apply the policies.
+
+   .. tab::
+
+      Workspace Config Policy
+
+      To set configuration policies at the workspace level:
+
+      #. Sign in to the Determined WebUI as a cluster or workspace administrator.
+      #. From your workspace, navigate to the **Config Policies** tab.
+      #. Choose **Experiments** or **Tasks** to display the editable configuration file.
+      #. Define the policies, then click **Apply**.
+      #. Confirm you want to apply the policies.
+
+CLI
+===
+
+Use the following commands to manage Task Config Policies via CLI:
+
+.. code:: bash
+
+   # List existing policies
+   det policy list
+
+   # Create a new policy
+   det policy create <policy_name> --config <policy_config_file.yaml>
+
+   # Update an existing policy
+   det policy update <policy_name> --config <updated_policy_config_file.yaml>
+
+   # Delete a policy
+   det policy delete <policy_name>
+
+*****************
+ Policy Examples
+*****************
+
+Below are some examples of Config Policies to help guide administrators.
+
+Limiting Resources
+==================
+
+Administrators can set constraints on resource usage, allowing them to manage how users allocate
+resources for workloads such as experiments, notebooks, tensorboards, shells, and commands. The two
+main configurable constraints are:
+
+-  ``resources.max_slots``: Limits the maximum number of slots (GPUs or CPUs) that can be used.
+-  ``resources.priority_limit``: Sets the priority limit for tasks.
+
+For Kubernetes resource managers, higher priority values indicate higher priority. For Agent
+resource managers, lower priority values indicate higher priority.
+
+**Example 1: Limiting GPU/CPU Slots and Priority for Experiments**
+
+The following example demonstrates how to set a maximum of 4 slots (GPUs or CPUs) and a priority
+limit of 15. This configuration applies to both Kubernetes and Agent resource managers, though the
+priority system behaves differently across the two:
+
+-  In Kubernetes, higher priority values indicate higher priority.
+-  In Agent resource managers, lower priority values indicate higher priority.
+
+.. code:: yaml
+
+   constraints:
+     resources:
+       max_slots: 4
+       priority_limit: 15
+
+In this example: - Users cannot set ``max_slots`` greater than 4. - Users cannot set a priority
+lower than 15 for Agent RMs or higher than 15 for Kubernetes RMs.
+
+**Example 2: Limiting Resources for Agent Resource Manager (RM)**
```

Comment:
How is this example specific to Agent Resource Manager?

Reply:
I think because of the term "cannot set a priority value lower than 15"  rather than "a user cannot set a priority value higher than 15"

I do think that we can abstract out the RM, give a more general constraint example, and then describe how behavior differs depending on the RM

Reply:
removing example 2 for now